test/ui/ResultActionsMenuTest.ts

```diff
@@ -0,0 +1,73 @@
+import * as Mock from '../MockEnvironment';
+import { ResultActionsMenu } from '../../src/ui/ResultActions/ResultActionsMenu';
+import { $$ } from '../../src/utils/Dom';
+
+export function ResultActionsMenuTest() {
+  describe('ResultActionsMenu', () => {
+    let test: Mock.IBasicComponentSetup<ResultActionsMenu>;
+
+    beforeEach(() => {
+      test = Mock.basicResultComponentSetup<ResultActionsMenu>(ResultActionsMenu);
+    });
+
+    afterEach(() => {
+      test = null;
+    });
+
+    it('should add show class when show is called', () => {
+      const component = $$(test.cmp.element);
+      expect(component.hasClass(ResultActionsMenu.SHOW_CLASS)).toBe(false);
```

Comment:
You could streamline the tests if you had a function called `componentHasShowClass`:
e.g.
```
function componentHasShowClass() {
    const component = $$(test.cmp.element);
    return component.hasClass(ResultActionsMenu.SHOW_CLASS);
}
```

This would allow you to cut the repetition in defining the `component` variable, calling `hasClass` on it, in the tests in this test suite.